lib/tools/node-modules/node-modules-dest-copy.ts

```diff
@@ -46,8 +46,8 @@ export class TnsModulesCopy {
 			shelljs.mkdir("-p", targetDir);
 			shelljs.cp("-Rf", dependency.directory, targetDir);
 
-			//remove platform-specific files (processed separately by plugin services)
-			const targetPackageDir = path.join(targetDir, dependency.name);
+			// remove platform-specific files (processed separately by plugin services)
+			const targetPackageDir = path.join(isScoped ? this.outputRoot : targetDir, dependency.name);
```

Comment:
Assuming we are looking at a scoped dependency, `dependency.name` will yield `@scope/module`, and `this.outputRoot` is `platforms/.../app/tns_modules` meaning the path to be removed on the following line will look like this: `app/tns_modules/@scope/module/platforms`. Or am I missing something?